docs/ides/overview.md

```diff
@@ -12,9 +12,8 @@ accurate compilation diagnostics in text editors and IDEs.
 There are two main IDE integrations:
 
 1. [IntelliJ](ides/intellij.md), the most popular Scala and Java IDE.
-1. [Metals](ides/metals.md), a work-in-progress Scala language server that
-   supports text editors such as Visual Studio Code, `vim`, Sublime Text and
-   Atom.
+1. [Metals](ides/metals.md), a Scala language server that supports text editors
```

Comment:
This change takes out the "work-in-progress". We made an explicit choice a while back to remove this. It might be best to align with the language here.

https://github.com/scalameta/metals/blob/f60048577e5da4e442af95f397273f7cac32ce37/website/blog/2019-06-28-thorium.md#no-more-work-in-progress